recipes/spyrmsd/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "spyrmsd" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 985861283513fcf9979c028ccd4a9c8dfb7771673a77a8182104de618950e261
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - numpy
+    - flit
+  run:
+    - python >=3.6
+    - {{ pin_compatible('numpy') }}
```

Comment:
I don't think you need the `pin_compatible`, since you aren't building against numpy in terms of a C-extension. Just `numpy` should be fine.